docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -54,15 +54,47 @@ Some container builds may use parent images from registries that require authent
 . Enter the password for the registry in **Password**.
 . Click **Add secret**.
 
+== Example of creating a quay.io secret
```

Comment:
nit: should this be still under the `== Creating registry pull secrets` section? 
Meaning that it may be better as a subsection like `.Creating a quay.io secret`.

Reply:
I thought that the quay.io is a registry this why I choose putting this example here , 
but I'm fine to move it to a subsection .

Reply:
I think it makes sense to make this a subsection because it is currently set as a separate section of the same level as the one above it, implying that it's separate from the registry pull secrets topic.

Reply:
Fixed